install.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
@Msarawan I think the name of the script should be more specific, like "install_olm.sh" so that it would be easy to get the idea what this script is about. Also the script is better to be moved into the **hack** dir.

Reply:
modified the changes.